MiscVBAFunctions/Modules/MiscCollection.bas

```diff
@@ -107,6 +107,29 @@ Public Function IsValueInCollection(ColInput As Collection, Val As Variant, Opti
 End Function
 
 
+Public Function IsKeyInCollection(Col As Collection, Key As Variant) As Boolean
```

Comment:
similar logic is in hasKey(), but this is possibly slightly more efficient due to being specific to collections

Reply:
Ah, shucks, I tried to make sure that I don't write any duplicate code. But if you think this is better for collections, we can leave it there.